app/models/payment.rb

```diff
@@ -42,6 +42,7 @@ def payment_date
 
   def resolution
     return fee_pay_info['paymentTypeDescription'] if fee_pay_info['paymentTypeDescription']
+    return fee_pay_info['paymentTypeID'] if fee_pay_info['paymentTypeID']
```

Comment:
I'm assuming that we always have a paymentTypeID, so do we do anything with it (translate?)? And does the final condition ever get called (return Paid)?

Reply:
We refactored the payment model.